.github/scripts/check_plugin.sh

```diff
@@ -1,5 +1,26 @@
 #!/bin/bash
 
+###### Information ############################################################################
+# Name:          check_plugin.sh
+# Maintainer:    ODFE Infra Team
+# Language:      Shell
+#
+# About:         This script is to check plugin artifacts availability for odfe distros
+#                It will prepare emails to odfe-release-process@amazon.com and a chime msg
+#                to inform user the status of the list of plugins
+#
+#                RED:    plugin artifact is NOT in S3 and corresponding repo has NO tag cut
+#                YELLOW: plugin artifact is NOT in S3 but corresponding repo has A tag cut
+#                GREEN:  plugin artifact is in S3
+#
+# Usage:         ./check_plugin.sh $PLUGIN_TYPES [$ODFE_VERSION]
+#                $PLUGIN_TYPES: zip,deb,rpm,kibana,clidvr,perftop (optional)
+#                $ODFE_VERSION: x.y.z (optional)
+#
+# Starting Date: 2020-05-29
+# Modified Date: 2020-07-30
+###############################################################################################
+
```

Comment:
This is awesome, thank you. I would remove the two date fields - Git tells us when the script was created and modified so there's no need to record that information twice. You could also remove the "maintainer" field, since everything in this repo is maintained by the infra team.

Reply:
This is resolved as we had a discussion.